modules/core/shared/src/main/scala/lucuma/core/util/Enumerated.scala

```diff
@@ -44,11 +45,12 @@ object Enumerated {
       def tag(a: A): String = a.productPrefix
     }
 
+  def fromTag[A](implicit ev: Enumerated[A]): Prism[String, A] =
```

Comment:
Nice, were you able to test it?

Reply:
The only thing I don't like here is that we have a `fromTag` on the trait. too. And the meaning of the 2 methods is quite different.